Toluene/MittalSung_2007_RCM_Tc_1022K.yaml

```diff
@@ -0,0 +1,160 @@
+---
+file-authors:
+    - name: Morgan Mayer
+      ORCID: 0000-0001-7137-5721
+file-version: 0
+chemked-version: 0.4.1
+reference:
+    doi: 10.1016/j.combustflame.2007.04.014
+    authors:
+        - name: Gaurav Mittal
+        - name: Chih-Jen Sung
+    journal: Combustion and Flame
+    year: 2007
+    volume: 150
+    pages: 355-368
+    detail: Toluene, Tc=1022K, phi=1
+experiment-type: ignition delay
+apparatus:
+    kind: rapid compression machine
+    institution: Case Western Reserve University
+    facility: CWRU RCM
+datapoints:
+    - temperature:
+        - 297 kelvin
+      ignition-delay:
+        - 17.1 ms
+      pressure:
+        - 907 torr
+      composition:
+        kind: mole fraction
+        species:
+          - species-name: toluene
+            InChI: 1S/C7H8/c1-7-5-3-2-4-6-7/h2-6H,1H3
+            amount:
+              - 0.00962
+          - species-name: O2
+            InChI: 1S/O2/c1-2
+            amount:
+              - 0.08654
+          - species-name: N2
+            InChI: 1S/N2/c1-2
+            amount:
+              - 0.10096
+          - species-name: Ar
+            InChI: 1S/Ar
+            amount:
+              - 0.80288
+      ignition-type:
+        target: pressure
+        type: d/dt max
+      compression-time:
```

Comment:
Can you use the new `rcm-data` field here: https://pr-omethe-us.github.io/PyKED/schema-docs.html#rcm-data-keys This also applies to replacing the `volume-history` with `time-histories`. See the example in the documentation: https://pr-omethe-us.github.io/PyKED/ck-tutorial.html#single-data-point-with-volume-history

Reply:
That will also let you add fields for Tc and Pc